package.json

```diff
@@ -33,8 +34,8 @@
     "cryptr": "^6.3.0",
     "date-fns": "^3.3.1",
     "date-fns-tz": "^3.1.3",
-    "isomorphic-dompurify": "^2.14.0",
     "iso-639-3": "^3.0.1",
+    "isomorphic-dompurify": "^2.14.0",
```

Comment:
we have this lib already in our package, I have added it a week ago.

Reply:
it was just moved by prettier I didn't change it